qcodes/utils/validators.py

```diff
@@ -193,7 +196,7 @@ def __init__(self, min_length: int = 0,
                             'no smaller than min_length')
         self._valid_values = ('.' * min_length,)
 
-    def validate(self, value: str, context: str = ''):
+    def validate(self, value: str, context: str = '') -> None:
```

Comment:
why do we type the value in validate methods to the type we expect? isn't the point that the type of value is always Any?

Reply:
that makes sense